code/modules/assembly/pass.dm

```diff
@@ -0,0 +1,36 @@
+/obj/item/assembly/timer/pass
+	name = "Personal Alert Safety System"
+	desc = "A device that plays a very loud alarm if it isn't moved for 30 seconds."
+	icon_state = "timer"
+	var/lastloc = null
+	var/loca = null
```

Comment:
Renaming this to current_turf or something would be easier for other people to comprehend